tensorflow/contrib/makefile/compile_ios_protobuf.sh

```diff
@@ -15,9 +15,10 @@
 # ==============================================================================
 # Builds protobuf 3 for iOS.
 
-set -x
 set -e
 
+export MACOSX_DEPLOYMENT_TARGET=$(sw_vers | grep ProductVersion | awk '{print $NF}')
```

Comment:
This can be a string literal like "10.12", or, as written above, it gets the version of the underlying macOS, making it the most compatible solution and future proofing the script.

As an example, running `sw_vers` on my computer I get:
```bash
$ sw_vers
ProductName:	Mac OS X
ProductVersion:	10.12.5
BuildVersion:	16F73
```